src/components/CreateNewPin.js

```diff
@@ -36,8 +194,16 @@ function CreateNewPin() {
           <UploadNewPinImageInput
             placeholder='Upload your Pin Image'
             type='file'
-            accept='image/gif, image/jpeg, image/png'></UploadNewPinImageInput>
+            accept='image/gif, image/jpeg, image/png, image/webp'
+            onChange={handleImageUpload}></UploadNewPinImageInput>
+          {selectedFile && <PreviewImage src={preview} />}
         </UploadNewPinImageLabel>
+        <CreatePinButton
+          onClick={() => {
+            uploadPinImage();
```

Comment:
Why not just 'onClick={uploadPinImage}?

Reply:
src/ components/ CreateNewPin.js line:226